### PR TITLE
#2357 - Replace separator component and vertical-divider class with v-divider component

### PR DESCRIPTION
**Pull request by sh16011993**
As a part of this PR, the following tasks were completed:

- [x] Replace separator component and vertical-divider class with v-divider component

**Screenshots:**

image

image

### CODE REVIEW

File: sources/packages/web/src/components/aest/institution/ApplicationOfferingChangeDetailsHeader.vue [hunk 1]
```diff
@@ -9,15 +9,25 @@
             : '-'
         "
       />
-      <separator v-if="headerDetails.institutionName"></separator>
+      <v-divider
```
Comment (ann-aot) [inline sources/packages/web/src/components/aest/institution/ApplicationOfferingChangeDetailsHeader.vue:12]: Very minor, In figma the color is `$brand-grey`. Divider has a property called color and the brand-grey theme is `secondary` in our application. so, you can add `<v-divider color="secondary"`

Comment (ann-aot) [reply sources/packages/web/src/components/aest/institution/ApplicationOfferingChangeDetailsHeader.vue:12]: Maybe we can go with this default opacity for vertical, as opacity:1  was too dark for  the vertical divider, when I was doing my story

Comment (sh16011993) [reply sources/packages/web/src/components/aest/institution/ApplicationOfferingChangeDetailsHeader.vue:12]: updated

**Review by ann-aot — APPROVED** [review]
👍 Thanks for doing the changes:

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for doing the changes, looks good 👍

**Review by dheepak-aot — APPROVED** [review]
Looks Good 👍

**Pull request merged**